### ISSUE TITLE
[FEATURE] gui for /team list

### ISSUE THREAD

**Issue by NOMASMP** [booksaw/BetterTeams#547]
### What is the feature request for?

it would be cool if it has gui thing with player heads of team owner players can just click their heads to join and see how many players in there with balance and etc 

### Feature requests are not going to be implemented by booksaw, instead you will have to find another developer to make a pull request to implement the feature.

- [X] I understand

**Comment by EarthCow** [comment]
Thanks for the feature request. This definitely would be nice to have. I'd be interested in doing this but I don't know when I'll have the time for something so large or if booksaw wants me doing something so large at all. Possible duplicate of #132

**Comment by booksaw** [comment]
yeah, I would like it added if you had time anything GUI based makes the plugin more accessible to players, 
I did some work a year or so back to create an API for GUIs, but I have not had the time to do anything with it at the moment.
